tests/test_consumer.py

```diff
@@ -6,13 +6,13 @@
 from datadog import statsd
 from six.moves import range
 
-from base import BaseTest, FakeBatchingKafkaConsumer, FakeWorker, FakeKafkaMessage, FakeKafkaProducer
+from base import BaseEventsTest, FakeBatchingKafkaConsumer, FakeWorker, FakeKafkaMessage, FakeKafkaProducer
 
 from snuba import processor
 from snuba.consumer import ConsumerWorker
 
 
-class TestConsumer(BaseTest):
+class TestConsumer(BaseEventsTest):
```

Comment:
This should not be event specific.
Is it possible to allow the test class to provide in a protected method or thorugh initialization the dataset name (when needed) without any other dataset specific customization when such customizaiton are not necessary for the test? 

Asking this because, going on, we will have more and more dataset agnostic tests, I would like to avoid creating a new dataset test class when not needed.

Reply:
These tests (as well as for processor, writer) are still coupled pretty hard to events. I pulled out a `get_event` method to avoid inheriting from BaseEventsTest. Or were you thinking to rewrite these to use a fake dataset?